app/src/resources/camera/usePreviewImage.ts

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react'
+
+import { useCapturePreviewImage } from '@opentrons/react-api-client'
+
+import { MIME_TYPES } from '../dataFiles/constants'
+
+import type { CameraImageSettings } from '@opentrons/api-client'
+
+export interface UsePreviewImageResult {
+  imgPath: string | undefined
+  isLoading: boolean
+  takePhoto: () => void
+}
+
+export function usePreviewImage(
+  settings: CameraImageSettings
+): UsePreviewImageResult {
+  const { data, refetch, isFetching } = useCapturePreviewImage(settings, {
+    responseType: 'blob',
+  })
```

Comment:
```suggestion
  }, {onSuccess..., onError..., onSettled...)
```
Unfortunately the `isFetching` isn't expressive enough to capture loading state without the UI looking blippy, so we need custom `isLoading` state. The callback methods you had before these changes looked good with the couple feedback comments, so we can just use those here!